api/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:17-alpine
+
+WORKDIR /app
+
+COPY . .
+
+RUN npm install
+
+EXPOSE 4000
+# required for docker desktop port mapping
```

Comment:
this is somethingr elated to docker desktop doesn't matter in case of cli

Reply:
There is a typo, it should **something related**